install/01_enabling_pg_stat_statements.mdx

```diff
@@ -21,12 +21,10 @@ On your database server, make sure that the extensions package is installed.
 
 Debian/Ubuntu:
 
-```
-# Replace 9.X with your installed Postgres version:
-sudo apt-get install postgresql-contrib-9.X
-```
+Debian/Ubuntu includes modules like `pg_stat_statements` in the main package,
+therefore no additional install is required.
 
-RedHat/CentOS/SLES:
+RedHat/CentOS:
```

Comment:
I confirmed that this is still required for RHEL. I removed SLES there since we don't mention anywhere else (also I don't think SLES uses yum, looks like it uses zypper).